.travis.yml

```diff
@@ -1,15 +1,14 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
-    - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
It was building on mac for all python versions: https://travis-ci.org/conda-forge/bob-feedstock/builds/163419304 unless some packages are deleted I don't think some dependencies are missing.